### PR TITLE
Objects JS Home work 6

### PR DESCRIPTION
**Pull request by AndreyTests**

### CODE REVIEW

File: objects/task5.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const users = [
+    {name : "John", email : "john@gmail.com", age : 25},
+    {name : "Stefan", email : "stefan@gmail.com", age : 30},
+    {name : "Daniel", email : "daniel@gmail.com", age : 32}
+]
+
+for (const newUser of users) {
+    const { name, email, age } = users;
```
Comment (sttaran) [inline objects/task5.js]: Apply destructuring to newUser but not to users array.

File: objects/task5.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+    console.log("newName:", newUser.name);
```
Comment (sttaran) [inline objects/task5.js]: And then use destructured values here

File: objects/task5.js (hunk 1 above)
Comment (sttaran) [reply objects/task5.js]: By the way, why you name it 'newUser'? It makes me confused. Loop does not create new objects , it just makes iteration. Just 'user' would be better for this variable.

**Review by sttaran — CHANGES_REQUESTED** [review]

File: objects/task5.js (hunk 2 above)
Comment (AndreyTests) [reply objects/task5.js]: Solved remark about destructured value.

File: objects/task5.js (hunk 1 above)
Comment (AndreyTests) [reply objects/task5.js]: Solved all remarks.

File: objects/task5.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const users = [
+    {name : "John", email : "john@gmail.com", age : 25},
+    {name : "Stefan", email : "stefan@gmail.com", age : 30},
+    {name : "Daniel", email : "daniel@gmail.com", age : 32}
+]
+
+for (const user of users) {
+    const { name, email, age } = user;
+
+    console.log("newName:", user.name);
```
Comment (sttaran) [inline objects/task5.js:10]: use desctructured values
```js
	console.log("newName:", name);
	console.log("newEmail:", email);
	console.log("newAge:", age);
```

**Review by sttaran — CHANGES_REQUESTED** [review]

**Pull request merged**